R/onLoad.R

```diff
@@ -1,21 +0,0 @@
-.onLoad <- function(libname, pkgname) {
-
-  if (is.null(getOption("rsconnect.max.bundle.size"))) {
-    options(rsconnect.max.bundle.size = 3145728000)
-  }
-
-  if (is.null(getOption("rsconnect.max.bundle.files"))) {
-    options(rsconnect.max.bundle.files = 10000)
-  }
-
-  # HTTP requests can take longer on Windows, so set a larger timeout
-  # by default (primarily used by the IDE when validating server URLs)
-  if (is.null(getOption("rsconnect.http.timeout"))) {
-    windows <- identical(.Platform$OS.type, "windows")
-    options(rsconnect.http.timeout = ifelse(windows, 20, 10))
-  }
-
-  if (is.null(getOption("rsconnect.metadata.sync.hours"))) {
-    options(rsconnect.metadata.sync.hours = 24)
```

Comment:
Already defaults to 24 in https://github.com/rstudio/rsconnect/blob/fb4ec22eee8c310a6936d5d513d0447099fbec1b/R/applications.R#L263